app/controllers/PromoLandingPage.scala

```diff
@@ -18,7 +18,7 @@ object PromoLandingPage extends Controller {
       case _ => None
     }
 
-  def render(promoCodeStr: String) = GoogleAuthenticatedStaffAction { implicit request =>
+  def render(promoCodeStr: String) = NoCacheAction { implicit request =>
```

Comment:
I think we should be caching this (if it's cached for no more than a day then that's within acceptable bounds for disabling the promo once it's no longer valid)